### DISCUSSION TITLE
Wake-on-LAN

### DISCUSSION THREAD

**Discussion by ojaksch** [ipxe/ipxe#286] [Ideas]
What about the idea for a new command which sends a Wake-on-LAN / MagicPacket?

**Comment by mcb30** [comment]
What would be the use case for sending that from within iPXE (rather than from within an operating system)?

**Reply by ojaksch** [reply]
To wake up a PC/Server at the **beginning** of the boot process instead of booting the OS, waking up and waiting for the (backup) server.
I'm totally fine with the latter, but my thought and idea was to send the WOL packet at a very earlier time to continue the service  of the booted OS without waiting for other things.

**Reply by NiKiZe** [reply]
This makes it the 3rd time then, https://forum.ipxe.org/showthread.php?tid=10023

**Reply by mcb30** [reply]
The existing `ping` command could possibly be extended to handle this, by using a broadcast or multicast destination address and allowing a WoL payload to be specified in place of the existing fixed payload.
